lib/sensors/pressure.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+#include <optional>
+
+#include <core/logger.hpp>
+#include <io/adc.hpp>
+
+namespace hyped::sensors {
+// TODOLater: test with hardware once we have it
+class SuspensionPressure {
+ public:
+  static std::optional<SuspensionPressure> create(core::ILogger &logger,
+                                                  std::shared_ptr<io::IAdc> adc);

```

Comment:
I think leave a comment for create stating that the adc instance passed to it "needs to be configured with pin X" (TODOLater: confirm X with electronics) -> this is just so that we rmb to check this later on in case we aren't told